src/ado2gh/Commands/LockRepoCommand.cs

```diff
@@ -64,8 +64,12 @@ public async Task Invoke(string adoOrg, string adoTeamProject, string adoRepo, s
 
             var ado = _adoApiFactory.Create(adoPat);
 
-            var teamProjectId = await ado.GetTeamProjectId(adoOrg, adoTeamProject);
-            var repoId = await ado.GetRepoId(adoOrg, adoTeamProject, adoRepo);
+            var teamProjectIdTask = ado.GetTeamProjectId(adoOrg, adoTeamProject);
+            var repoIdTask = ado.GetRepoId(adoOrg, adoTeamProject, adoRepo);
+
+            var teamProjectId = await teamProjectIdTask;
+            var repoId = await repoIdTask;
+
```

Comment:
This code is going to exactly run the same as previous code. If you want to make them run in parallel you should use `Task.WhenAll()`:
```C#
await Task.WhenAll(new[] {  teamProjectIdTask, teamProjectIdTask };
```
Now that being said, I don't think we need to micro-optimize this code unless we have a very strong reason to do so. Making things unnecessarily parallel usually will makes the code base harder to maintain.